certified-connectors/Nextcom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,585 @@
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "contactId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Use this if you want to force an update to an existing contact. If this isn't supplied, normal duplicate checking rules will be applied",
+                  "title": "Contact ID"
+                },
+                "contactIdSettings": {
+                  "type": "string",
+                  "description": "If contactId is applied, should the existing contact be moved to the new list? ",
+                  "title": "Contact ID settings",
+                  "enum": [
+                    "MaintainInList",
+                    "MoveToList"
+                  ]
+                },
+                "userId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The user who created the contact ",
+                  "title": "User ID"
+                },
+                "contactGroup": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "1 = Private person, 2 = Company",
+                  "title": "Contact Group"
+                },
+                "listId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The list the contact will be added or moved to",
+                  "title": "List ID"
+                },
+                "duplicateSettings": {
+                  "type": "string",
+                  "description": "Select what happens upon duplicate detection",
+                  "title": "Duplicate settings",
+                  "enum": [
+                    "DontImport",
+                    "ResetAndMove",
+                    "Move",
+                    "ResetAndMoveWithForceList",
+                    "MoveWithForceList",
+                    "ResetAndMoveAndSetDateTimeCreated",
+                    "ResetAndMoveAndSetDateTimeCreatedIndependent",
+                    "AllowAllDuplicates",
+                    "AllowAllDuplicatesWithLogging"
+                  ]
+                },
+                "forceFromListId": {
+                  "type": "array",
+                  "title": "Force from list ID",
+                  "items": {
+                    "type": "integer",
+                    "format": "int32"
+                  },
+                  "description": "Used in conjunction with duplicateSettings. If DuplicateSetting 4 is chosen, duplicates will only be moved from the lists supplied in this parameter"
+                },
+                "orgNoOrSSN": {
+                  "type": "string",
+                  "description": "Org number for companies, social security number for private persons",
```

Comment:
@natalie-pienkowska and @ptalrejagithub, Is it safe to take input of SSN and similar highly confidential information from users thru Power Automate. Is it required to let our legal team know about this?

Reply:
Please make it secure string at least

Reply:
securestring is not a valid swagger 2.0 type.
I added a password-format, tag, as it is the closest I could think of. 
https://github.com/microsoft/PowerPlatformConnectors/pull/1123/commits/d290958dc36d64f2234cb310c1b59fcafed9bc98